components/notification/overview.md

```diff
@@ -120,6 +85,22 @@ The `NotificationModel` class is used to add new notifications to the page. You
 | `Icon`  | `string` | Specifies the icon that will render in the component if the `ShowIcon` parameter is set to `true`. You can find more information on adding an icon to a Telerik Component in [Telerik Font Icons article]({%slug general-information/font-icons%}#icon-in-telerik-component). |
 | `Text`  | `string` | the text that will be rendered in the Notification component. |
 
+### Styling and Appearance
+
+The Notification appearance and styling can be easily customized. You can change the notifications' colors, animation type, size and stack order. You can find more information in the [Notification Appearance]({%slug notification-appearance%}) article.
+
+## Notifications Reference and Methods
+
+To use the methods of the Blazor Notification, add a reference to the component instance.
```

Comment:
"add a reference to the component instance" - how? Consider a short snippet for this.